src/EcurrungException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ecurring\WooEcurring;
+
+use Exception;
+
+/**
+ * Super type for all app exceptions.
+ */
+class EcurrungException extends Exception
```

Comment:
Maybe there is a typo in the name of this class? EcurringException?

Reply:
Yep, it's a typo, thanks. Fixed.